frontend/src/components/studies/StudyDetails.tsx

```diff
@@ -58,6 +84,18 @@ const StudyDetails = ({studyId} : StudyDetailsProps) => {
                 <Descriptions.Item label="Project" span={2}>{project?.name ?? ''}</Descriptions.Item>
```

Comment:
Not very useful to display the project. Instead add the start and end steps of the study.